doc/overview/storage.md

```diff
@@ -15,7 +15,7 @@ storage capacity dedicated to each target) or by spanning more targets
 the unit of provisioning and isolation. DAOS pools cannot span across multiple 
 systems.
 
-A pool can host multiple transactional object store called DAOS containers. 
+A pool can host multiple transactional object stores called DAOS containers. 
```

Comment:
(style)  trailing whitespace

Reply:
the trailing whitespace is consistent with the rest of the file